todo/src/main.rs

```diff
@@ -33,7 +33,7 @@ async fn main() -> io::Result<()> {
     let app = move || {
         debug!("Constructing the App");
 
-        let templates: Tera = Tera::new("/templates/**/*").unwrap();
+        let templates: Tera = Tera::new("templates/**/*").unwrap();
```

Comment:
Use the cargo manifest relative path since it allows the example to be run from the root with `cargo run --bin actix-todo` (albeit without static files loading correctly).

```suggestion
        let templates: Tera = Tera::new(concat!(env!("CARGO_MANIFEST_DIR"), "/templates/**/*")).unwrap();
```

Reply:
Happy to make the change but `cargo run --bin actix-todo` panics either way. For it to run, it has to load `DATABASE_URL` from a .env file that isn't present at the top level of the project.

Reply:
DATABASE_URL can be imput as a regular env var as well, eg `DATABASE_URL=x@localhost/db cargo run --bin actix-todo`. Maybe for other reasons it's best to leave it as is so folks running the example arent confused when part of it works and part doesnt.